### ISSUE TITLE
testcafe-reporter-dashboard and GHSA-hjrf-2m68-5959

### ISSUE THREAD

**Issue by gemal** [DevExpress/testcafe#7438]
### What is your Scenario?

testcafe-reporter-dashboard is affected by GHSA-hjrf-2m68-5959
https://github.com/advisories/GHSA-hjrf-2m68-5959

```
GHSA-hjrf-2m68-5959
fix available via `npm audit fix --force`
Will install testcafe@15.1.317922, which is a breaking change
node_modules/jsonwebtoken
  testcafe-reporter-dashboard  >=0.0.9
  Depends on vulnerable versions of jsonwebtoken
  node_modules/testcafe-reporter-dashboard
    testcafe  1.17.2-alpha.1 - 2.2.0-rc.1
    Depends on vulnerable versions of testcafe-reporter-dashboard
    node_modules/testcafe
```

### What is the Current behavior?

.

### What is the Expected behavior?

.

### What is your public website URL? (or attach your complete example)

.

### What is your TestCafe test code?

.

### Steps to Reproduce

1.
2.
3.
.

### TestCafe version

.

### Command-line arguments

.

**Comment by miherlosev** [comment]
Hi @gemal,

Thank you for reporting this vulnerability. We will fix it.

**Comment by AndreyBelym** [comment]
Fixed in `testcafe@2.2.0-rc.2`.

**Comment by codambro** [comment]
Still seeing this error in `testcafe@2.2.0`
